workers/unity/Assets/Fps/Scripts/SetupLogic/ClientWorkerConnector.cs

```diff
@@ -76,7 +74,7 @@ protected override string SelectLoginToken(List<LoginTokenDetails> loginTokens)
 
         protected override ConnectionService GetConnectionService()
         {
-            if (useSessionFlow)
+            if (!string.IsNullOrEmpty(Deployment))
```

Comment:
If we define a property

```
private shouldUseSessionFlow => !string.IsNullOrEmpty(Deployment);
```

it could help make this a bit more fluent when reading 😄